src/reducers/books.js

```diff
@@ -0,0 +1,18 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actionTypes';
+
+const initialState = { books: [{ id: Math.floor(Math.random() * 100), title: 'The Dark Tower', category: 'Horror' }, { id: Math.floor(Math.random() * 10), title: 'Fantastic Beast', category: 'Sci-Fi' }] };
```

Comment:
Well done guys,

According to the instruction on pathwright, you need to create the book object in `src/index.js`

![image](https://user-images.githubusercontent.com/31897434/110299622-3272d180-7ff6-11eb-9f48-0bac7376aca5.png)
  